build_all.bat

```diff
@@ -81,6 +81,8 @@ call group2\build.bat 2017.4.35f1
 call group2\build.bat 2017.4.36f1
 call group2\build.bat 2017.4.37f1
 call group2\build.bat 2017.4.38f1
+call group2\build.bat 2017.4.39f1
+call group2\build.bat 2047.4.40f1
```

Comment:
typo in version number - should be `2017.4.40f1`?

Reply:
Yes, I also noticed 208.4.23f1 instead of 2018.4.23f1 in extract.sh this morning. Fixes incoming.

Reply:
Pushed.